### ISSUE TITLE
Error tar-ing path containing spaces

### ISSUE THREAD

**Issue by Philoozushi** [swarajban/npm-cache#18]
Hi there,
If one has a path containing a space, e.g. `/Users/John/Projects/My project`, npm-cache cannot tar packages, like the exemple below.
```
[npm-cache] [INFO] [bower] archiving dependencies from /Users/John/Projects/My project
tar: could not chdir to '/Users/John/Projects/My'
[npm-cache] [ERROR] [bower] error tar-ing /Users/John/Projects/My project
[npm-cache] [ERROR] error installing dependencies
```

**Comment by swarajban** [comment]
Got it, should be an easy fix. I'll try to get to all these small issues in the next week or so

**Comment by swarajban** [comment]
Fixed in 0.4.0!
